File: src/main/java/um/haberes/core/model/dto/FormularioImprimir.java
package um.haberes.core.model.dto;

import lombok.AllArgsConstructor;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;
import um.haberes.core.model.ContactoEntity;
import um.haberes.core.model.LegajoControlEntity;

import java.util.List;

@Getter
@Setter
@EqualsAndHashCode
@ToString
@NoArgsConstructor
@AllArgsConstructor
public class FormularioImprimir {

    private List<LegajoControlEntity> legajoControls = null;

    private List<ContactoEntity> contactos = null;
}

File: src/main/java/um/haberes/core/kotlin/model/dto/FormularioImprimir.kt
package um.haberes.core.kotlin.model.dto

import um.haberes.core.kotlin.model.Contacto
import um.haberes.core.kotlin.model.LegajoControl

data class FormularioImprimir(

    var legajoControls: List<LegajoControl?>? = null,
    var contactos: List<Contacto>? = null

)
